src/main/java/duke/Ui.java

```diff
@@ -0,0 +1,119 @@
+package duke;
+
+import java.util.ArrayList;
+import java.util.Scanner;
+
+import duke.tasks.Task;
+
+/**
+ * User Interface class where interactions with user are handled
+ */
+public class Ui {
+    private Scanner input;
+
+    /**
+     * Initializes the Ui object
+     */
+    public Ui() { }
+
+    /**
+     * Shows a line to dictate separation
+     */
+    public static void showLine() {
+        System.out.println("____________________________________________________________ \n");
+    }
+
+    /**
+     * Displays a welcome message
+     *
+     * @param list in which the tasks are stored
+     */
+    public void welcomeMessage(TaskList list) {
```

Comment:
Maybe naming the method with a verb can help the program be more understandable?